contents/membership/_index/main.markdown

```diff
@@ -78,9 +70,9 @@ subscription.
 <input type="hidden" name="hosted_button_id" value="6YY74YT8UKT9E">
 <table>
 <tr><td><input type="hidden" name="on0" value="HackerspaceSG monthly membership">HackerspaceSG monthly membership</td></tr><tr><td><select name="os0">
-	<option value="Regular">Regular : $128.00 SGD - monthly</option>
-	<option value="Associate">Associate : $64.00 SGD - monthly</option>
-	<option value="Resident">Resident : $256.00 SGD - monthly</option>
+	<option value="Resident">Regular : $256.00 SGD - monthly</option>
+	<option value="Regular">Community : $128.00 SGD - monthly</option>
+	<option value="Associate">Concessionary : $64.00 SGD - monthly</option>
```

Comment:
Why have the option values dissociated from the text?

Reply:
The option values match the Paypal subscription names, I think

Reply:
What Bob said. I believe PayPal knows how much to charge for each subscription based on the value field, so they remain unchanged for now.